packages/babel-plugin-react-i18n/README.md

```diff
@@ -0,0 +1,61 @@
+# `@shopify/babel-plugin-react-i18n`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fbabel-plugin-react-i18n.svg)](https://badge.fury.io/js/%40shopify%2Fbabel-plugin-react-i18n.svg)
+
+A babel plugin to auto-fill withI18n arguments from an adjacent translations folder
+
+## Installation
+
+```bash
+$ yarn add @shopify/babel-plugin-react-i18n
+```
+
+## Usage
+
+Simply add `@shopify/react-i18n` to your list of babel plugins. For example, via `.babelrc`:
+
+```json
+{
+  "plugins": ["@shopify/react-i18n"]
```

Comment:
Does Babel not get confused with this and the actual package? Did you consider just throwing this in to the package itself?

Reply:
I thought babel plugins are always supposed to begin with `babel-plugin-*`, but I guess that's not the case?

Reply:
How would you throw this into the package? I guess using some sort of `@shopify/react-i18n/babel` suffix? Would babel know how to find that?

Reply:
You’d just use the full path to the module, yeah